doc/Advanced-configuration.md

```diff
@@ -319,6 +319,31 @@ An example configuration can look as follows:
 `{odbc_server, {pgsql, "localhost", "username", "database", "pass",
                [{ssl, required}, {ssl_opts, [{verify, verify_peer}, {cacertfile, "path/to/cacert.pem"}]}]}}.`
 
+### ODBC SSL connection setup
+
+If you've configured MongooseIM to use ODBC driver, i.e. you've provided ODBC connection string to `odbc_server` option, e.g.
+
+```erlang
+{odbc_server, "DSN=mydb"}.
+```
+
+then SSL options, along other connection options, should be present in `~/.odbc.ini` file.
+
+To enable SSL connection `sslmode` option needs to be set to `verify-full`.
+Additionally, you can provide path to the CA certificate using `sslrootcert` option.
```

Comment:
Additionally, you can provide the path to the CA certificate using the `sslrootcert` option.